Packs/DuoAdminApi/Integrations/DuoAdminApi/DuoAdminApi.py

```diff
@@ -330,6 +330,27 @@ def delete_u2f_token(token_id):
     demisto.results('Token with ID ' + token_id + ' deleted successfully')
 
 
+def set_user_status(username, status):
+    result = admin_api.get_users_by_name(username)
+    user_id = result[0]["user_id"]
+    res = admin_api.update_user(user_id, status=status)
+    demisto.results('User ' + username + 'was set to status ' + status)
+
+
+def get_duo_admin_by_name(name):
+    duo_administrators = admin_api.get_admins()
+    duo_administrator = next((admin for admin in duo_administrators if admin['name'] == name), None)
```

Comment:
if it's possible to get an admin ID using the `get_user_id(username)` function, please change the API call to `get_admin` (see ref: https://github.com/duosecurity/duo_client_python/blob/master/duo_client/admin.py#L2496)

```suggestion
    admin_id = get_user_id(name)
    duo_administrator = admin_api.get_admin(admin_id)
```